content/docs/components-and-props.md

```diff
@@ -15,26 +15,25 @@ redirect_from:
 prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
-
 Los componentes permiten separar la UI en piezas independientes, reutilizables y pensar en cada pieza de forma aislada.
 Esta página proporciona una introducción a la idea de los componentes.
 Puedes encontrar una [API detallada sobre componentes aqui](/docs/react-component.html).
 
 Conceptualmente, los componentes son como las funciones de JavaScript. Aceptan entradas arbitrarias (llamadas "props") y devuelven a React elementos que describen lo que debe aparecer en la pantalla.
 
-## Function and Class Components
+## Componentes Funcionales y de Clase
 
-The simplest way to define a component is to write a JavaScript function:
+La forma más sencilla de definir un componente es escribir una función de JavaScript:
 
 ```js
 function Welcome(props) {
   return <h1>Hello, {props.name}</h1>;
 }
 ```
 
-This function is a valid React component because it accepts a single "props" (which stands for properties) object argument with data and returns a React element. We call such components "function components" because they are literally JavaScript functions.
+Esta función es un componente de React válido porque acepta un solo argumento de objeto "props" (que significa propiedades) con datos y devuelve un elemento de React. Llamamos a dichos componentes "funcionales" porque literalmente son funciones JavaScript.
```

Comment:
I'd change `(que significa propiedades)` by`(que proviene de propiedades)`

Reply:
That sounds better